tensorflow/core/kernels/conv_ops_gpu_2.cu.cc

```diff
@@ -26,11 +26,15 @@ namespace tensorflow {
 
 typedef Eigen::GpuDevice GPUDevice;
 template struct functor::InflatePadAndShuffle<GPUDevice, float, 4, int>;
+#ifndef NVIDIA_TEGRA
 template struct functor::InflatePadAndShuffle<GPUDevice, float, 4,
                                               Eigen::DenseIndex>;
```

Comment:
Does Tegra not even support the compilation of DenseIndex?

Reply:
For some reason, nvcc recognize DenseIndex as int, causing a duplicate definition error.